README.md

```diff
@@ -92,6 +92,21 @@ pipeline {
 }
 ```
 
+### Detecting whether a build was started by the trigger in a script:
+```groovy
+def triggerCause = currentBuild.rawBuild.getCause(
+    org.jenkinsci.plugins.pipeline.github.trigger.IssueCommentCause
+)
+
+if (triggerCause) {
+    echo("Build was started by ${triggerCause.userLogin}, who wrote: " +
+         "\"${triggerCause.comment}\", which matches the " +
+         "\"${triggerCause.triggerPattern}\" trigger pattern.")
```

Comment:
@aaronjwhiteside, this might not actually work, because `IssueCommentCause` is currently not serializable. I just got an error:
```

an exception which occurred:
	in field com.cloudbees.groovy.cps.impl.BlockScopeEnv.locals
	...
	in object org.jenkinsci.plugins.workflow.cps.CpsThreadGroup@1ff0d85c
Caused: java.io.NotSerializableException: org.jenkinsci.plugins.pipeline.github.trigger.IssueCommentCause
	at org.jboss.marshalling.river.RiverMarshaller.doWriteObject(RiverMarshaller.java:860)
	at org.jboss.marshalling.river.BlockMarshaller.doWriteObject(BlockMarshaller.java:65)
        ...
```